nerfstudio reconstruction problem

Hi, 
when I run "python3 scripts/reconstruction_benchmark/main.py --config-file config/recon_benchmark.yaml" to get nerfstudio reconstruction result. I the the following error message: 
```
Starting Reconstruction Benchmark
Project folder: data/sequences/2024-03-13-observatory-quarter-01
Loading transform matrix from data/sequences/2024-03-13-observatory-quarter-01/T_gt_lidar.txt
Running 'nerfacto' on data/sequences/2024-03-13-observatory-quarter-01/outputs/nerfstudio/2024-03-13-observatory-quarter-01/transforms_metric.json
Traceback (most recent call last):
  File "/home/oxford_spires_dataset/scripts/reconstruction_benchmark/main.py", line 306, in <module>
    recon_benchmark.run_nerfstudio("nerfacto", json_filename="transforms_metric.json")
  File "/home/oxford_spires_dataset/scripts/reconstruction_benchmark/main.py", line 214, in run_nerfstudio
    metric_cloud_file = run_nerfstudio(ns_config, ns_data_config, export_cloud)
  File "/home/oxford_spires_dataset/scripts/reconstruction_benchmark/nerf.py", line 75, in run_nerfstudio
    update_argv(ns_config)
  File "home/oxford_spires_dataset/scripts/reconstruction_benchmark/nerf.py", line 62, in update_argv
    assert sys.argv[0].endswith(".py") and len(sys.argv) == 1, "No args should be provided for the script"
AssertionError: No args should be provided for the script
```
the content of config/recon_benchmark.yaml is 
```
reconstruction_benchmark:
  project_folder: "./data/sequences/2024-03-13-observatory-quarter-01"
  gt_folder: "./data/ground_truth_map/observatory-quarter"

  run_gt_cloud_processing: False
  run_lidar_cloud_processing: False
  run_lidar_cloud_evaluation: False
  run_colmap: False
  run_colmap_sim3: False
  run_mvs: False
  run_mvs_evaluation: False
  run_nerfstudio: True
  run_novel_view_synthesis_only: False
```

Can you help to solve this problem?

Thanks

--- a/scripts/reconstruction_benchmark/nerf.py
+++ b/scripts/reconstruction_benchmark/nerf.py
@@ -59,7 +59,12 @@ def create_nerfstudio_dir(colmap_dir, ns_dir, image_dir):
 
 def update_argv(nerfstudio_config, follow_up=False):
     if not follow_up:
-        assert sys.argv[0].endswith(".py") and len(sys.argv) == 1, "No args should be provided for the script"
+        assert sys.argv[0].endswith(".py")
+        if sys.argv[0].endswith("main.py"):
+            assert len(sys.argv) <= 3, "Only the config file should be provided."
+            assert sys.argv[1] == "--config-file"
+            sys.argv = [sys.argv[0]]
+        assert len(sys.argv) == 1, "No args should be provided for the script"
     for k, v in nerfstudio_config.items():
         if k in ("method", "dataparser"):
             sys.argv.append(f"{v}")

--- a/scripts/reconstruction_benchmark/nerf.py
+++ b/scripts/reconstruction_benchmark/nerf.py
@@ -62,8 +62,8 @@ def update_argv(nerfstudio_config, follow_up=False):
         assert sys.argv[0].endswith(".py")
         if sys.argv[0].endswith("main.py"):
             assert len(sys.argv) <= 3, "Only the config file should be provided."
-            assert sys.argv[1] == "--config-file"
-            sys.argv = [sys.argv[0]]
+            if len(sys.argv) >= 2 and sys.argv[1] == "--config-file":
+                sys.argv = [sys.argv[0]]
         assert len(sys.argv) == 1, "No args should be provided for the script"
     for k, v in nerfstudio_config.items():
         if k in ("method", "dataparser"):

